src/include/txn.i

```diff
@@ -975,13 +975,7 @@ __wt_txn_read(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt, WT_ITEM *key, uint
     }
 
     /* If the start time point is visible then we need to return the ondisk value. */
-    if (F_ISSET(session, WT_SESSION_HS_IGNORE_VISIBILITY) ||
```

Comment:
Can we simplify the condition at lines 863 and 955 with this change?

Reply:
91e8023